Defra.Cdp.Backend.Api/Services/Github/RepositoryService.cs

```diff
@@ -65,13 +65,18 @@ await Collection
 
     public async Task<List<Repository>> AllRepositories(bool excludeTemplates, CancellationToken cancellationToken)
     {
+        var builder = Builders<Repository>.Filter;
+        var filter = builder.Empty;
+        var withoutTemplatesFilter = builder.Eq(r => r.IsTemplate, false);
+        var withCdpTopicFilter = builder.Where(r => r.Topics.Contains("cdp"));
```

Comment:
I struggled a little with making this check case insensitive. Mongo did not like 

```
builder.Where(r => r.Topics.Contains("cdp", System.OrdinalIgnoreCaseComparer));
```